### PR TITLE
Remove rs2-19h1 theme resources from CBS package

### PR DESCRIPTION
**Pull request by licanhua**
resources.pri is ~16M, and after I removed rs2-19h1 theme resources from CBS package, the size reduced to ~1.8M.

The procedure is like this:
1. generate CBS\CBSResources.pri which have Microsoft.UI.Xaml.CBS namespace
2. dump the CBSResources.pri to .pri.xml with `/dt detailed`
3. set the value to " "(not "") for all rs2-19h1 theme resouces.
4. repack the changed file into CBSResources.pri
5. dump the CBS\CBSResources.pri and CBSResources.pri without `/dt` flag, the two dump doesn't include the embeded data, so they should be the same. also CBS\CBSResources.pri and CBSResources.pri should not be the same.

### CODE REVIEW

File: build/FrameworkPackage/MakeFrameworkPackage.ps1 [hunk 1]
```diff
@@ -29,6 +29,90 @@ function Copy-IntoNewDirectory {
...
+{
+    Param($WindowsSdkBinDir, $makePriArgs)
+    $makepriNew = "`"" + (Join-Path $WindowsSdkBinDir "makepri.exe") + "`" $makePriArgs"
+    Write-Host $makepriNew
+    cmd /c $makepriNew
+    if ($LastExitCode -ne 0) { Exit 1 }
+}
+
+# only keep 21h1 themeresources, and set othe themeresources to " "
+# I didn't set it to "" because "" will change the `makepri dump` layout, but " " will not.
```
Comment (jevansaks) [inline build/FrameworkPackage/MakeFrameworkPackage.ps1:42]: So you don't remove the downlevel entries, you just make them empty?

**Review by jevansaks — APPROVED** [review]

File: build/FrameworkPackage/MakeFrameworkPackage.ps1 (hunk 1 above)
Comment (licanhua) [reply build/FrameworkPackage/MakeFrameworkPackage.ps1:42]: yes. To be exactly, the long Base64Value is set to IA==

```
<NamedResource name="rs2_generic_2dot5.xbf" index="18" uri="ms-resource://Microsoft.UI.Xaml.CBS/Files/Microsoft.UI.Xaml/Themes/rs2_generic_2dot5.xbf">
	<Decision index="1">
		<QualifierSet index="0" />
	</Decision>
	<Candidate type="EmbeddedData">
		<QualifierSet index="0" />
		<Base64Value>IA==</Base64Value>
	</Candidate>
</NamedResource>
```

Comment (licanhua) [reply build/FrameworkPackage/MakeFrameworkPackage.ps1:42]: Here is the size comparison:
image

**Pull request merged**